core/src/test/java/org/apache/iceberg/rest/responses/TestCreateNamespaceResponse.java

```diff
@@ -101,7 +101,8 @@ public void testDeserializeInvalidResponse() {
     String jsonMisspelledKeys = "{\"namepsace\":[\"accounting\",\"tax\"],\"propertiezzzz\":{\"owner\":\"Hank\"}}";
     AssertHelpers.assertThrows(
         "A JSON response with the keys spelled incorrectly should fail to deserialize and validate",
-        JsonProcessingException.class,
+        IllegalArgumentException.class,
+        "Invalid namespace: null",
```

Comment:
Because `RESTObjectMapper` will ignore unknown fields, the exception thrown in these tests where the keys are misspelled turn out to be more specific.

I've updated them to use the proper thrown exception type, as well as add part of the exception message to help clarify the tests purpose and intent.